examples/simple-username-database/index.js

```diff
@@ -0,0 +1,70 @@
+
+// one of the most simplest example of javascript using supabase datbase to take input from user and save it in database.
+
+const { createClient } = require("@supabase/supabase-js");
+const readline = require("readline");

```

Comment:
Should we update to modern ES Module imports?

Reply:
@preetsuthar17 thoughts on these other comments? I see we are still using `require()`.